bibliographies/ch10.bib

```diff
@@ -0,0 +1,481 @@
+	number = {6},
+	journal = {Journal of Southern African Institute of Mining and Metallurgy},
+	author = {Krige, D.G.},
+	year = {1951},
+	keywords = {application of lognormal curve, application of statistics, frequency distribution, gold mining, gold values, kriging, mine valuation, ore valuation, statistical theory and Witwatersrand},
+	pages = {119--139},
+}
+
+@article{goovaerts_kriging_2008,
+	title = {Kriging and {Semivariogram} {Deconvolution} in the {Presence} of {Irregular} {Geographical} {Units}},
+	volume = {40},
+	issn = {15378276},
+	url = {https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3624763/pdf/nihms412728.pdf},
+	abstract = {Conduction time is typically ignored in computational models of neural network function. Here we consider the effects of conduction delays on the synchrony of neuronal activity and neural oscillators, and evaluate the consequences of allowing conduction velocity (CV) to be regulated adaptively. We propose that CV variation, mediated by myelin, could provide an important mechanism of activity-dependent nervous system plasticity. Even small changes in CV, resulting from small changes in myelin thickness or nodal structure, could have profound effects on neuronal network function in terms of spike-time arrival, oscillation frequency, oscillator coupling, and propagation of brain waves. For example, a conduction delay of 5 ms could change interactions of two coupled oscillators at the upper end of the gamma frequency range (â¼100 Hz) from constructive to destructive interference; delays smaller than 1 ms could change the phase by 30Â°, significantly affecting signal amplitude. Myelin plasticity, as another form of activity- dependent plasticity, is relevant not only to nervous system development but also to complex information processing tasks that involve coupling and synchrony among different brain rhythms. We use coupled oscillator models with time delays to explore the importance of adaptive time delays and adaptive synaptic strengths. The impairment of activity-dependent myelination and the loss of adaptive time delays may contribute to disorders where hyper- and hypo-synchrony of neuronal firing leads to dysfunction (e.g., dyslexia, schizophrenia, epilepsy)},
+	number = {1},
+	journal = {Mathematical Geology},
+	author = {Goovaerts, Pierre},
+	year = {2008},
+	pmid = {1000000221},
+	note = {ISBN: 6176321972
+\_eprint: NIHMS150003},
```

Comment:
Looks like there's a missing `},` on the line above, and I'm not sure if the `\_` is intentional.

Reply:
Looks like this is a Zotero artifact. The page it scraped from must have added additional information into the "notes" section, so this is actually one line (and it should work), but it's not necessary so I removed it!.